src/GeocodeSharp/RequestBuilderBase.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Net;
+using GeocodeSharp.Google;
+
+namespace GeocodeSharp
+{
+    public abstract class RequestBuilderBase
+    {
+        protected string _clientKey;
+        protected UsageMode _mode;
+        protected string _clientId;
+        protected string _cryptoKey;
+
+        protected string _domain = "https://maps.googleapis.com";
+        protected const string _apiPath = "/maps/api/geocode/json?";
+
+        /// <param name="domain"></param>
+        /// <param name="isProtected">only needed so that signature doesnt collide with another ctor</param>
+        protected RequestBuilderBase(string domain, bool isProtected = true)

```

Comment:
Yep, it's there just for protected ctor not to collide with public ctor that takes one string parameter